.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+aliases:
+  - &restore_sbt_cache
+    key: sbt-{{ checksum "/tmp/scala_version" }}-cache
+
+  - &save_sbt_cache
+    key: sbt-{{ checksum "/tmp/scala_version" }}-cache-{{ epoch }}
+    paths:
+      - "~/.ivy2/cache"
+      - "~/.sbt"
+      - "~/cache/coursier"
+
+  - &run_cibuild
+    - checkout
+    - run: echo "${SCALA_VERSION}" > /tmp/scala_version
+    - restore_cache: *restore_sbt_cache
+    - run:
+        name: Executing cibuild
+        command: ./scripts/cibuild
+    - save_cache: *save_sbt_cache
+
+version: 2
+workflows:
+  version: 2
+  build:
+    jobs:
+      - "openjdk8-scala2.11.12-nodelts"
+      - "openjdk8-scala2.12.8-nodelts"
+
+jobs:
+  "openjdk8-scala2.11.12-nodelts":
+    docker:
+      - image: circleci/openjdk:8-stretch-node
+    environment:
+      SCALA_VERSION: 2.11.12
```

Comment:
Is there a way that we can specify the Scala version without the PATCH version? It would be great to be able to do 2.11 vs. 2.11.x, and have it self resolve to the latest patch release.

Reply:
There wasn't an immediately obvious way. 

I tried passing the version as `$ ./sbt ++2.12.* test` and `$ ./sbt ++2.12.+ test` but it translates those values as string literals. I believe this corresponds to the `scalaVersion` value in `build.sbt`.